src/meta/node/src/lib.rs

```diff
@@ -192,6 +193,10 @@ pub struct MetaNodeOpts {
     #[override_opts(path = system.license_key)]
     pub license_key: Option<LicenseKey>,
 
+    /// The path of the license key file to be watched and hot-reloaded.
+    #[clap(long, env = "RW_LICENSE_KEY_PATH")]
```

Comment:
typo found 🤣

Reply:
Discussed with @BugenZhao elsewhere, here's the conclusion: `RW_LICENSE_KEY_PATH` will be kept instead of being changed to the `RW_LICENSE_KEY_FILE` in the PR description.